packages/now-cli/src/commands/inspect.js

```diff
@@ -314,6 +190,9 @@ function stateString(s) {
     case 'ERROR':
       return chalk.red(s);
 
+    case 'BUILDING':
+      return chalk.grey(s);
+
```

Comment:
We're missing `QUEUED` and `CANCELED`.

```
  readyState:
    | 'BUILDING'
    | 'ERROR'
    | 'INITIALIZING'
    | 'QUEUED'
    | 'READY'
    | 'CANCELED';
```

Reply:
I think the default option on line 200 should be `return chalk.gray(s || 'UNKNOWN');`